.gitignore

```diff
@@ -15,10 +16,13 @@
 # Temporary development stuff (Manual tests, etc.)
 /dev
 test.env
+.env
 
 # Python cruft
 *.pyc
 .python-version
 
 # C extensions
 *.pyd
+
+playtest_custom.py
```

Comment:
You can put your own test files under the `dev` directory without a special case in .gitignore

Reply:
done